views/chooseAll-answer.pug

```diff
@@ -0,0 +1,45 @@
+if answerForm
+	.form-group.col.s12
+		each index in [1,2,3,4]
```

Comment:
this should be a globally defined number within the `chooseAll-answer.js` file. this is so that this number can be updated easily. In here you can do something like 
``` pug
each index in Array(variable)
```

Reply:
I have tried, It doesn't work, instead of array I used a for loop to make it look clean

Reply:
I have used it in the past, please try again